src/tests/unit/tests/DetailsView/components/target-page-changed-view.test.tsx

```diff
@@ -6,38 +6,32 @@ import { DisplayableVisualizationTypeData } from '../../../../../common/configs/
 import { VisualizationType } from '../../../../../common/types/visualization-type';
 import { TargetPageChangedView, TargetPageChangedViewProps } from '../../../../../DetailsView/components/target-page-changed-view';
 
-type RenderTestCases = {
-    title?: string;
-    toggleLabel?: string;
-    subtitle?: JSX.Element;
-};
-
 describe('TargetPageChangedView', () => {
-    describe('renders', () => {
-        const clickHandlerStub: () => void = () => {};
-        const testCases: RenderTestCases[] = [
-            {},
-            { title: 'title' },
-            { toggleLabel: 'toggle-label' },
-            { subtitle: <span>subtitle</span> },
-            { title: 'title', toggleLabel: 'toggle-label', subtitle: <span>subtitle</span> },
-        ];
+    it('renders without optional subtitle', () => {
+        testRenderWithOptionalSubtitle(undefined);
+    });
 
-        it.each(testCases)('case %o', testCase => {
-            const { title = '', toggleLabel = '' } = testCase;
+    it('renders with optional subtitle', () => {
+        testRenderWithOptionalSubtitle(<span>test subtitle content</span>);
```

Comment:
why not use `it.each` for this 2 cases?

Reply:
The rendering of the JSX.Element in the test title was the thing causing the node 10/12 incompatibility; there didn't seem to be a good option for pretty-printing it reasonably, it was either "print it as an object in a way that is platform specific" or "print it as '[Object]'".

I could have just made the content text be the test case discriminator and had the test body use a conditional to wrap it (`const subtitle = testCaseSubtitleText ? <span>{testCaseSubtitleText}</span> : undefined;`), but my default instinct is always to avoid conditionals in test cases where it's straightforward to do so :)